pkg/webhook/webhook.go

```diff
@@ -48,11 +58,41 @@ const (
 	FleetWebhookCfgName      = "fleet-validating-webhook-configuration"
 	FleetWebhookSvcName      = "fleetwebhook"
 
-	crdResourceName           = "customresourcedefinitions"
-	memberClusterResourceName = "memberclusters"
-	namespaceResouceName      = "namespaces"
-	replicaSetResourceName    = "replicasets"
-	podResourceName           = "pods"
+	crdResourceName                      = "customresourcedefinitions"
+	bindingResourceName                  = "bindings"
+	configMapResourceName                = "configmaps"
+	endPointResourceName                 = "endpoints"
+	eventResourceName                    = "events"
+	limitRangeResourceName               = "limitranges"
+	persistentVolumeClaimsName           = "persistentvolumeclaims"
+	podTemplateResourceName              = "podtemplates"
```

Comment:
I am not sure what this is

Reply:
It is a namespaced resource 
![image](https://github.com/Azure/fleet/assets/17449572/38bb3b44-5662-4fcc-83b5-09e9c4f01ca0)